proposed/2021/BulkRestoreCoordination.md

```diff
@@ -127,11 +138,33 @@ The proposal is that each time NuGet determines a restore might need run, it che
     }
 ```
 
+#### Implementation details
+
+- Given that there are risks with this proposal, the NuGet implementation will have a means of disabling this feature.
+- NuGet will only account for projects that have registered an IVsProjectRestoreInfoSource. A PackageReference based project does not have to register an IVsProjectRestoreInfoSource.
+- At solution load time, NuGet should be able to remove the sliding window, and rely on reporting information to the customer about potentially delayed restores. The frequency and the means of reporting can be determined at a later point.
+- In any non solution load scenario, NuGet can loop through all IVsProjectRestoreInfoSource objects, calling `HasPendingNomination`. When the first project reports `true` NuGet will call `WhenNominated`. There is no need for NuGet to call `WhenNominated` on all pending projects.

```

Comment:
When the first project reports `true` NuGet will call `WhenNominated` _on that project and any other project_?

Reply:
I made it more explicit. NuGet will call `WhenNominated` on that project only.